lib/features/taskcluster_proxy.js

```diff
@@ -33,10 +33,16 @@ export default class TaskclusterProxy {
 
     var cmd = [
         '--client-id=' + task.runtime.taskcluster.clientId,
-        '--access-token=' + task.runtime.taskcluster.accessToken,
+        '--access-token=' + task.runtime.taskcluster.accessToken
+    ];
+    // only pass in a certificate if one has been set
+    if (task.runtime.taskcluster.certificate && task.runtime.taskcluster.certificate.length > 0) {
+        cmd = cmd.concat('--certificate=' + task.runtime.taskcluster.certificate);
```

Comment:
can use cmd.push() instead as I think concat creates a new array and returns it, where as you could just callpush and it will append it to the existing array.